python/tops2vrt.py

```diff
@@ -92,115 +102,73 @@ def getLinePixelBbox(geobbox, latFile, lonFile):
     '''
     Main driver.
     '''
-    
-    ##Parse command line
+
+    # Parse command line
     inps = cmdLineParse()
 
-    ###Get ann list and slc list
-    slclist = glob.glob(os.path.join(inps.indir,'SLC','*','*.slc.full'))
+    # Get a list of co-registered full SLC VRTs
+    slclist = glob.glob(os.path.join(inps.indir, 'SLC', '*', '*.slc.full.vrt'))
```

Comment:
How about making "SLC" and ".slc.full.vart" also input arguments with default values as we have here?
In principle any slc that is readable by GDAL should work down the line with fringe core modules.